lighthouse-core/report/html/templates.html

```diff
@@ -783,24 +783,64 @@
 
 <template id="tmpl-lh-3p-filter">
   <style>
+    /* The input checkbox element is invisible by should span the same width and height
+       as the label. Flex ensures the input is the correct width, while the transform
+       ensures that the input is in the correct position. */
```

Comment:
```suggestion
       ensures that the label is in the correct position. */
```

is this what it's supposed to be?

just curious why this approach and not `position: absolute; left: 0; top: 0; right: 0; bottom: 0` on the input? I probably skimmed or forgot the discussion first time around :)

the `translateY` on the label (that's also 1px off from the filter with margin-bottom?) is just what sticks out to me here

Reply:
thanks ... I was having trouble getting the input to not take the width of the page. Guess I forgot to `0` all the sides..